kairon/api/app/routers/bot/data.py

```
@@ -369,6 +370,32 @@
         event.enqueue()
     return {"message": "Document content upload in progress! Check logs."}
 
+@router.post("/upload/{type}/{collection_name}", response_model=Response)
+async def upload_file_content(
+    file_content: UploadFile,
+    type: Text = Path(description="Upload class type"),
```

Ruff (0.12.2)

Do not perform function call `Path` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)